### ISSUE TITLE
6X50 Hashrates

### ISSUE THREAD

**Issue by johnnyde94** [todxx/teamredminer#611]
Hello, I wanted to know if there are any plans in the work to increase 6X50 cards hash rate. I have a 6750 and am only getting 40mh when the card should be able to hit 50 easily. I know there is speculation it has to do with loose mem timings but I also heard there was a similar issue with the 6700 cards on launch. 

My Card is still in its return window and I am wondering if that is the way to go or am I better off waiting it out. I use HiveOS for my rig and I know that hive is still reporting the cards as a 6700. 

Basically, what are the odds the hash rate will improve over time? I am hoping there are miner optimizations that can be done to help with the hash rate or bios mods (know that wouldn't be you) or mem timing tweaks something that can be done to save these cards. 

Thanks Again for all of the hard work!

**Comment by CamJava007** [comment]
From absolutely every tech geek that I follow... the overall sentiment has been that the 6X50 cards sadly hash either the same or slower than their predecessors.  My 6700 XT gets 47, while the 6750 XT only gets about 42... 😕

**Comment by grenelt** [comment]
I'm not really an expert, but i think the advantages of the 6x50 gpus are (partially) result of advanced caching optimized for gaming - which struggles at mining. So there is in my opinion no way to speed them up.

**Comment by CamJava007** [comment]
According to literally every tech geek I have watched, read and listened to... you are 💯 correct.

**Comment by grenelt** [comment]
... and i'm watching and reading no tech geeks at all! So presumeable i am one! ;)
